opnieuw/retries.py

```diff
@@ -115,15 +106,11 @@ def __init__(
             max_calls_total, retry_window_after_first_call_in_seconds
         )
 
-    def __iter__(self) -> Iterator[Action]:
+    def __iter__(self) -> Iterator[DoWait]:
```

Comment:
And I would add a docstring to explain what you are doing here now.

Reply:
Or as a top docstring.

Reply:
Added!